spec/graphql/types/repository/permissions_type_spec.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Types::Repository::PermissionsType do
+  let!(:repository) { create :repository_compound }
+  let!(:user) { create :user }
+
+  subject { repository }
+
+  let(:permissions_type) do
+    OntohubBackendSchema.types['RepositoryPermissions']
+  end
+
+  before do
+    subject.add_member(user, 'admin')
```

Comment:
Please test for something other than 'admin' at this point. The current test would also allow an implementation that constantly returns 'admin'.